From ee483f763e1d7480d75056cd909a7fc7b8f9644b Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Thu, 6 Sep 2018 12:29:56 -0700
Subject: [PATCH 0821/1239] octeontx: board and drivers: use correct location
 for header files

This is due to change in location of header file in upstream
code for errno.h and libfdt.h.

Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 board/cavium/octeontx/fdt.c        | 2 +-
 board/cavium/octeontx/octeontx.c   | 2 +-
 board/cavium/octeontx2/fdt.c       | 2 +-
 board/cavium/octeontx2/octeontx2.c | 2 +-
 drivers/mmc/octeontx_mmc.c         | 2 +-
 drivers/mtd/nand/octeontx_bch.c    | 2 +-
 drivers/mtd/nand/octeontx_nand.c   | 2 +-
 drivers/net/cavium/octeontx_bgx.c  | 4 ++--
 drivers/net/cavium/octeontx_xcv.c  | 4 ++--
 9 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/board/cavium/octeontx/fdt.c b/board/cavium/octeontx/fdt.c
index a0c660cd9d..b54399d45a 100644
--- a/board/cavium/octeontx/fdt.c
+++ b/board/cavium/octeontx/fdt.c
@@ -10,7 +10,7 @@
 #include <environment.h>
 #include <asm/io.h>
 #include <linux/compiler.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 #include <fdtdec.h>
 #include <fdt_support.h>
 #include <asm/arch/atf.h>
diff --git a/board/cavium/octeontx/octeontx.c b/board/cavium/octeontx/octeontx.c
index 4f1472fa7d..5457665897 100644
--- a/board/cavium/octeontx/octeontx.c
+++ b/board/cavium/octeontx/octeontx.c
@@ -10,7 +10,7 @@
 #include <netdev.h>
 #include <asm/io.h>
 #include <linux/compiler.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 #include <fdt_support.h>
 #include <asm/arch/octeontx.h>
 #include <asm/arch/atf.h>
diff --git a/board/cavium/octeontx2/fdt.c b/board/cavium/octeontx2/fdt.c
index 61e778472e..69281af6b7 100644
--- a/board/cavium/octeontx2/fdt.c
+++ b/board/cavium/octeontx2/fdt.c
@@ -10,7 +10,7 @@
 #include <environment.h>
 #include <asm/io.h>
 #include <linux/compiler.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 #include <fdtdec.h>
 #include <fdt_support.h>
 #include <asm/arch/atf.h>
diff --git a/board/cavium/octeontx2/octeontx2.c b/board/cavium/octeontx2/octeontx2.c
index dcb442ab6f..ce977d5eda 100644
--- a/board/cavium/octeontx2/octeontx2.c
+++ b/board/cavium/octeontx2/octeontx2.c
@@ -11,7 +11,7 @@
 #include <errno.h>
 #include <asm/io.h>
 #include <linux/compiler.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 #include <fdt_support.h>
 #include <asm/arch/octeontx2.h>
 #include <asm/arch/atf.h>
diff --git a/drivers/mmc/octeontx_mmc.c b/drivers/mmc/octeontx_mmc.c
index 7ea797da7f..b29cb7bdba 100644
--- a/drivers/mmc/octeontx_mmc.c
+++ b/drivers/mmc/octeontx_mmc.c
@@ -40,7 +40,7 @@
 #include "mmc_private.h"
 #include <errno.h>
 #include <pci.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 #include <fdtdec.h>
 #include <asm/arch/fdt-helper.h>
 #include <asm/io.h>
diff --git a/drivers/mtd/nand/octeontx_bch.c b/drivers/mtd/nand/octeontx_bch.c
index 85a3463650..5a24811260 100644
--- a/drivers/mtd/nand/octeontx_bch.c
+++ b/drivers/mtd/nand/octeontx_bch.c
@@ -6,7 +6,6 @@
  */
 
 #include <common.h>
-#include <libfdt.h>
 #include <malloc.h>
 #include <memalign.h>
 #include <pci.h>
@@ -18,6 +17,7 @@
 #include <linux/errno.h>
 #include <linux/err.h>
 #include <linux/ioport.h>
+#include <linux/libfdt.h>
 #include <linux/mtd/mtd.h>
 #include <linux/mtd/nand_bch.h>
 #include <linux/mtd/nand_ecc.h>
diff --git a/drivers/mtd/nand/octeontx_nand.c b/drivers/mtd/nand/octeontx_nand.c
index 4ab205d4c0..33e295019f 100644
--- a/drivers/mtd/nand/octeontx_nand.c
+++ b/drivers/mtd/nand/octeontx_nand.c
@@ -6,7 +6,6 @@
  */
 
 #include <common.h>
-#include <libfdt.h>
 #include <malloc.h>
 #include <memalign.h>
 #include <pci.h>
@@ -19,6 +18,7 @@
 #include <linux/errno.h>
 #include <linux/err.h>
 #include <linux/ioport.h>
+#include <linux/libfdt.h>
 #include <linux/mtd/mtd.h>
 #include <linux/mtd/nand_bch.h>
 #include <linux/mtd/nand_ecc.h>
diff --git a/drivers/net/cavium/octeontx_bgx.c b/drivers/net/cavium/octeontx_bgx.c
index 1339e8f0a0..bbb17b80e6 100644
--- a/drivers/net/cavium/octeontx_bgx.c
+++ b/drivers/net/cavium/octeontx_bgx.c
@@ -9,6 +9,7 @@
 
 #include <config.h>
 #include <common.h>
+#include <errno.h>
 #include <net.h>
 #include <dm.h>
 #include <pci.h>
@@ -17,10 +18,9 @@
 #include <malloc.h>
 #include <miiphy.h>
 #include <asm/io.h>
-#include <asm/errno.h>
 
 #ifdef CONFIG_OF_LIBFDT
- #include <libfdt.h>
+ #include <linux/libfdt.h>
  #include <fdt_support.h>
 #endif
 
diff --git a/drivers/net/cavium/octeontx_xcv.c b/drivers/net/cavium/octeontx_xcv.c
index ff299b84cf..d9c166f42e 100644
--- a/drivers/net/cavium/octeontx_xcv.c
+++ b/drivers/net/cavium/octeontx_xcv.c
@@ -9,6 +9,7 @@
 
 #include <config.h>
 #include <common.h>
+#include <errno.h>
 #include <net.h>
 #include <dm.h>
 #include <pci.h>
@@ -17,10 +18,9 @@
 #include <malloc.h>
 #include <miiphy.h>
 #include <asm/io.h>
-#include <asm/errno.h>
 
 #ifdef CONFIG_OF_LIBFDT
- #include <libfdt.h>
+ #include <linux/libfdt.h>
  #include <fdt_support.h>
 #endif
 
-- 
2.29.0

